src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapter.cs

```diff
@@ -371,6 +371,84 @@ static public int RemainingTimeAsVisible
             }
         }
 
+        /// <summary>
+        /// Enables the local Bluetooth adapter, asynchronously.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Enable()
+        {
+            int ret = BluetoothAdapterImpl.Instance.Enable();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to Enable - " + (BluetoothError)ret);
+                BluetoothErrorFactory.ThrowBluetoothException(ret);
+            }
+        }
+
+        /// <summary>
+        /// Disables the local Bluetooth adapter, asynchronously.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Disable()
+        {
+            int ret = BluetoothAdapterImpl.Instance.Disable();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to Disable - " + (BluetoothError)ret);
+                BluetoothErrorFactory.ThrowBluetoothException(ret);
+            }
+        }
+
+        /// <summary>
+        /// Sets the visibility mode.
+        /// </summary>
+        /// <param name="visible">The Bluetooth visibility mode to set. true is Discoverable, false is NonDiscoverable.</param>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void SetVisibility(bool visible)
+        {
+            if (IsBluetoothEnabled)
+            {
+                int ret = BluetoothAdapterImpl.Instance.SetVisibility(visible == true ? VisibilityMode.Discoverable : VisibilityMode.NonDiscoverable, 0);
+                if (ret != (int)BluetoothError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to SetVisibility - " + (BluetoothError)ret);
+                    BluetoothErrorFactory.ThrowBluetoothException(ret);
+                }
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+
+        }
+
+        /// <summary>
+        /// Sets the visibility mode.
+        /// </summary>
+        /// <param name="timeout">The timeout until the visibility mode is changed NonDiscoverable(in seconds) from Discoverable.</param>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void SetVisibility(int timeout)
```

Comment:
How about chagne name to `EnableDiscoverable`

``` c#
public static void EnableDiscoverable(int duration);
public static void EnableDiscoverable();

public static void DisableDiscoverable();
```

Reply:
Done. Thank you.

Reply:
Done. Thank  you.